docs/development/accessibility-process.md

```diff
@@ -0,0 +1,244 @@
+    * Issues can be reported on automatically testable violations of guidelines.
+    * Issues can be reported on violations of accessibility rules listed in a KS
+      entry, but also the on the rules themselves if they need improvement.
+      The KS entry and implementation are amended via the normal KS process.
+    * Issues can be reported against soft / non-objective parts of the accessibility
+      guidelines. These require discussion and decisions. Accessibility Guidelines
+      are to be amended or clarified to become more objective. Tickets of this kind
+      are assigned to the UI/UX/A11y expert group.
+
+* [Jour Fixe](https://docu.ilias.de/goto.php?target=wiki_1357_Jour_Fixe_Agendas)
+    * Maintainers can put forward their Accessibility issues that came up during
+      implementation as "Development Issues".
+    * Can be contacted for resolving discussions/deciding on issues among different stakeholders on A11y questions.      
+
+* [UI Components](../..//src/UI/README.md)
+    * Lists UI Components of ILIAS along with a purpose description an A11y on specific UI Components.
+    * Can be used to lookup of the mechanism of specific parts of the UI.
+
+* Accessibility Mailing List
+    * First POC for Maintainers
+    * Maintainers, writers of Feature wiki articles can turn themselves to Acessibility
+      Mailing List to get their issues looked at.
+    * The mailing list is managed by the UI/UX/A11y expert group
+
+## Activities in Release Cycle
+To above goals are not achievable in a single step. They need to be tackled continuously in an iterative process aiming to push 
+closer towards achieving those goals with each iteration. This section shows how we understand our release cycle as one
+iteration in the process of enhancing A11y in ILIAS. The main responsibility for following those steps listed in this 
+cycle resides by the Technical Board supported by the SIG A11y and the A11y expert group.
+
+### Contracted External Beta Accessibility Assessment - November
+* Early in Beta phase a formative Accessibility Assessment is carried out by a registered auditor.
+* Those audits have the main goal of evaluate the A11y of a aspect of ILIAS (e.g. new Features, status of
+  a given service module or component or guidelines).
+* Those audits provide feedback on the quality of the tested aspect. 
+* Note, certification of any kind or generating lists for issue trackers are NOT goals of that assessment.
+* The Product Manager signs the contract with the registered auditor. The SIG may help in providing the funding.
+* ILIAS society provides test installation.
+
+### Huge Project Identification Jour Fixe / Yearly Project
+* During the Huge Project Identification Meeting we try to identify large UI heavy components to make sure accessibility 
+issues are addressed early and continuously. 
+* Huge UI heavy components may choose to contract external accessibility testing before merging.
+
+### After-Action Review Accessibility - March
```

Comment:
Dieser Block war für mich ganz schwer verständlich: Was gehört in das After-Action Review als Sitzung und was gehört da hin als Prozess. Ich habe einfach einmal versucht den Text in eine Richtung gerade zu ziehen.